### PR TITLE
CLIP-1699: Update versions for 2.2.0 release.

### PR DESCRIPTION
**Pull request by tarka**
## Pull request description

Minor release as it contains a slight behavioural change (bidirectional PV)

_Provide description for the PR_

## Checklist
- [ ] I have successful end to end tests run (with & without domain)
- [ ] I have added unit tests (if applicable)
- [/] I have user documentation (if applicable)

### CODE REVIEW

**Review by bianchi2 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Change Log
 
+## 2.2.0
```
Comment (nghazali) [inline CHANGELOG.md:3]: This is a bug fix release, the release number should be `2.1.1`

**Review by nghazali — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (tarka) [reply CHANGELOG.md:3]: The bidirectional PV feels more like a minor functional change. @bianchi2 what do you think?

Comment (nghazali) [reply CHANGELOG.md:3]: Still this is a bug fix. This fixes the problem of not removing pv in uninstall in some cases.

Comment (bianchi2) [reply CHANGELOG.md:3]: It's just a smarter way to create volumes - we make sure that the volume we create for a DC app does not get bound to elastic search (or any other 3rd party claim)

**Pull request merged**